pkg/apis/hive/v1/syncset_types.go

```diff
@@ -259,6 +245,20 @@ type SyncSetSpec struct {
 	ClusterDeploymentRefs []corev1.LocalObjectReference `json:"clusterDeploymentRefs"`
 }
 
+// SyncSetCommonStatus is common status for SyncSets and SelectorSyncSets
+type SyncSetCommonStatus struct {
+	// ObservedGeneration is the generation of the syncset that has been observed by the syncset controller.
+	ObservedGeneration int64 `json:"observedGeneration"`
+
+	// Resources is the list of resources effected by the syncset.
```

Comment:
effected seemed right to me (https://www.diffen.com/difference/Affected_vs_Effected#:~:text=%22Affected%22%20means%20%22impacted%2C,brought%20about%2C%20produced%20something.%22&text=The%20BP%20oil%20spill%20adversely,of%20Mexico%20and%20surrounding%20areas.)

Reply:
Getting into grammatical weeds here, one could argue for "effected" if we were only ever creating resources, but even that would be a bit odd. Since we're creating, changing, or deleting them, "affected" is correct. If you really wanted to use "effect" it would be something like "...the list of resources on which the syncset effects changes."

Totally not going to die on a hill for this. Feel free to ignore.

Reply:
+1 for affected